# Make sure the configure script doesn't fail over a 'no version information available' warning
# by wpoely86@gmail.com
diff -ur cairo-1.12.18.orig/build/aclocal.cairo.m4 cairo-1.12.18/build/aclocal.cairo.m4
--- cairo-1.12.18.orig/build/aclocal.cairo.m4	2013-03-15 21:29:27.000000000 +0100
+++ cairo-1.12.18/build/aclocal.cairo.m4	2015-01-14 14:08:01.000000000 +0100
@@ -101,9 +101,9 @@
 	$1
 	AC_LINK_IFELSE(
 		[AC_LANG_SOURCE([$_compile_program])],
-		[cairo_cc_stderr=`test -f conftest.err && cat conftest.err`
+		[cairo_cc_stderr=`test -f conftest.err && cat conftest.err | grep -v 'no version information available' `
 		 cairo_cc_flag=yes],
-		[cairo_cc_stderr=`test -f conftest.err && cat conftest.err`
+		[cairo_cc_stderr=`test -f conftest.err && cat conftest.err | grep -v 'no version information available' `
 		 cairo_cc_flag=no])
 
 	if test "x$cairo_cc_stderr" != "x"; then
